src/EditorFeatures/CSharpTest/ChangeSignature/ReorderParametersTests.cs

```diff
@@ -265,7 +265,8 @@ public static void M(this C goo, int y, int x, string c = ""test_c"", string b =
     { }
 }";
 
-            await TestChangeSignatureViaCommandAsync(LanguageNames.CSharp, markup, updatedSignature: permutation, expectedUpdatedInvocationDocumentCode: updatedCode);
+            await TestChangeSignatureViaCommandAsync(LanguageNames.CSharp, markup, updatedSignature: permutation,
+                expectedUpdatedInvocationDocumentCode: updatedCode, expectedSelectedIndex: 0);
```

Comment:
:memo: although the parameter config has `0` for the `SelectedIndex`, the UI dialog will make an adjustment and select parameter `y` instead before the `this` parameter cannot be moved or removed.